lib/Db/User.php

```diff
@@ -23,27 +23,30 @@
 
 namespace OCA\Deck\Db;
 
-use OCP\IUser;
+use OCP\IUserManager;
 
 class User extends RelationalObject {
```

Comment:
I'd love it if we could get that RelationalObject goodness into server at some point, seems very useful!

Reply:
I'm not too happy anymore with the approach and think it should actually be decoupled from the entity itself. I'm more in favour of having separate Model classes in the future like Talk has.